autonight/autonight.c

```diff
@@ -165,7 +178,209 @@ int main(int argc, char *argv[]) {
 
         return 0;
 }
+#define ISP_INFO  "/proc/jz/isp/isp_info"
+#define EXPOSURE  "ISP exposure log2 id: "
+#define IRIDIX    "ISP Iridix strength : "
```

Comment:
Just curious, what does that iridix strenght actually mean?

Reply:
BTW https://github.com/Dafang-Hacks/drivers/blob/16770a7c7f67520b85f8120adf9e0035357e7c91/isp/tx-isp/apical-isp/tx-isp-core.c#L2258 is the only source I can find to explain iridix and the rest of the parameters.